rllib/tests/test_execution.py

```diff
@@ -63,12 +63,19 @@ def step(self, action):
 class MockBaseEnv(_VectorEnvToBaseEnv):
     def __init__(self, vector_env, slow):
         super().__init__(vector_env)
-        self.slow = slow
+        import logging
+        self.logger = logging.getLogger()
```

Comment:
What is this logger being used for?

Reply:
Nothing, it was for debugging and I forgot to delete it. Thanks. Removed